Source/SvgDocument.cs

```diff
@@ -206,7 +206,9 @@ public static void EnsureSystemIsGdiPlusCapable()
         {
             try
             {
+#if !NETSTANDARD20
                 using (var matrix = new Matrix(0f, 0f, 0f, 0f, 0f, 0f)) { }
+#endif

```

Comment:
@gvheertum - this removes the libgdiplus check for netstandard2, do you think this is a problem, or can be solved another way?

Reply:
This check fails on netstandard2.0 and UWP (and probably other not supported netstandard2 targets). I have tested on UWP with Svg.Skia and removing this check enables Svg.Skia to work and System.Drawing (libgdiplus ) is not supported anyway on UWP. For now this is best solution I have found.

Reply:
I have removed if/def from EnsureSystemIsGdiPlusCapable and instead added it to Open method.

Reply:
I'm wondering what happens if a Matrix is encountered in that case. What kind of error did you get that made you decide to remove the check? If the Matrix cannot be loaded I expect issues lateron when Matrix related stuff is encountered.

Reply:
I am not using System.Drawing part of SVG so I don't need Matrix and other classes from gdi. This check prevented me from using SVG when using netstandard2.0 version on platforms like UWP. Util we have splitted drawing from object model processing that's only solution I think.

Reply:
Fair point, however this will break all other cases where people are using .NET Standard with graphics enabled. This will re-open the issues that caused us to put the check in (the exception that was thrown was not that clear). So honestly I am not a big fan of disabling the check for .NET standard as a whole.

Perhaps we can add a static flag somewhere in the SVG lib that you can set to false to disable the check? In that case we can also put warnings in the xml doc of the flag. Something like Svg.Configuration.SkipGDIPlusCapabilityCheck. In that case we will keep the check in the normal use-case and will be able to skip it when explicitly set.

Reply:
> static flag

I will submit PR with static flag proposal.